### PR TITLE
Supertuxkart Example on Agones Site

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix
> /kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Kalaiselvi84** [comment]
@igooch, Could you please review the changes and provide your feedback? I'm uncertain if there's anything else that needs to be included.

File: site/content/en/docs/Examples/supertuxkart.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+publishDate:
+description: >
+  This Supertuxkart example shows how to set up, deploy, and manage a Supertuxkart game server on a Kubernetes cluster using Agones. It highlights an approach to integrate with existing dedicated game servers.
+---
+
+## Prerequisite
+
+To get started, ensure the following prerequisites are met:
+
+  - You have a running Kubernetes cluster.
```
Comment (markmandel) [inline site/content/en/docs/Examples/supertuxkart.md:14]: You'll need the SuperTuxKart client 😃

File: site/content/en/docs/Examples/supertuxkart.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+title: "Deploying and Running SuperTuxKart Server Using Agones"
+linkTitle: "Supertuxkart"
```
Comment (markmandel) [inline site/content/en/docs/Examples/supertuxkart.md:3]:
```suggestion
linkTitle: "SuperTuxKart"
```

File: site/content/en/docs/Examples/supertuxkart.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+**Navigate to Online Play**: From the main menu, select the "Online" option and then select "Enter server address" from the available options.
+
+**Enter Server Details**: In the subsequent screen, you will be prompted to input the IP address and port number in order to join the game. Please enter the IP address and port number obtained from the `kubectl get gameservers` command.
+
+![enter ip and port](../../../images/supertuxkart-enter-ip-port.png)
+
+**Join the Game**: After entering the server details, proceed to join the server. You should now be connected to your Agones-managed SuperTuxKart game server and ready to play.
+
+![start race](../../../images/supertuxkart-start-race.png)
```
Comment (markmandel) [inline site/content/en/docs/Examples/supertuxkart.md:82]: We can also connect some AI bots!

I do a demo of this here:
https://youtu.be/ACJvwZBwWew?list=PLqqp1QEhKwa60zuzZej8-va3Q_Za6bchq&t=1027

Documentation here: https://github.com/supertuxkart/stk-code/blob/master/NETWORKING.md#testing-server

So you can have other players - can we please add this too!

Comment (Kalaiselvi84) [reply site/content/en/docs/Examples/supertuxkart.md:84]: Information on playing with AI bots has been included. Please take a moment to review it and share your thoughts.

File: site/content/en/docs/Examples/supertuxkart.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+```bash
+kubectl logs -f <supertuxkart-game-server-pod-name>
+```
+
+## Connect to the Game Server
+
+After allocating a GameServer from the fleet and obtaining its status and IP, you're ready to connect and play. Here’s how to use the server IP and port to join the game with the SuperTuxKart client:
+
+**Launch SuperTuxKart**: Start the SuperTuxKart client you downloaded earlier by running the executable for your operating system ([documentation]https://supertuxkart.net/FAQ)
```
Comment (markmandel) [inline site/content/en/docs/Examples/supertuxkart.md:74]: Markdown issue
```suggestion
**Launch SuperTuxKart**: Start the SuperTuxKart client you downloaded earlier by running the executable for your operating system ([documentation](https://supertuxkart.net/FAQ))
```

I think will fix it. If you need local preview: `make site-server`

**Review by markmandel — COMMENTED** [review]
Found one rendering issue, but otherwise, this is good to go.

File: site/content/en/docs/Examples/supertuxkart.md (hunk 4 above)
Comment (Kalaiselvi84) [reply site/content/en/docs/Examples/supertuxkart.md]: I've fixed the link issue

**Comment by Kalaiselvi84** [comment]
I've removed the `viewing the game server log` section from the documentation. Please let me know if anything missing..

**Review by markmandel — APPROVED** [review]

**Pull request merged**